Dockerfile

```diff
@@ -26,16 +26,33 @@ RUN npm run build
 
 FROM alpine
 LABEL org.opencontainers.image.title="Copacetic" \
-    org.opencontainers.image.description="Copacetic Extension for Docker Desktop" \
-    org.opencontainers.image.vendor="Microsoft" \
+    org.opencontainers.image.description="Directly patch container images given the vulnerability scanning results from popular tools like Trivy." \
+    org.opencontainers.image.vendor="Project Copacetic" \
     com.docker.desktop.extension.api.version="0.3.4" \
-    com.docker.extension.screenshots="" \
+    com.docker.extension.screenshots="\
+    [ \
+    {\"alt\": \"Copa Light Screenshot\", \"url\": \"https://raw.githubusercontent.com/jgrer/copa-extension/dockerfile-label-documentation/.github/images/demo1.png\"}, \
```

Comment:
reminder to update your fork before merge

Reply:
we were thinking of doing this in the next PR so that they are already hosted on the copa org repo